### PR TITLE
Add Table of Contents to README

### PR DESCRIPTION
**Pull request by akostibas**
## Description

The README was getting super long and it was hard to find things. Adding an auto-generated table of contents.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,6 +6,33 @@ This repository contains the application source code for the Personal Property P
 
 This prototype was built by a [Defense Digital Service](https://www.dds.mil/) team in support of USTRANSCOM's mission.
 
+## Table of Contents
+
+<!-- toc -->
```
Comment (brainsik) [inline README.md]: Can you add or move the comment about how to regenerate up here? Putting it at the end seems like a great way to miss it until after you've manually modified things.

Comment (akostibas) [reply README.md]: Unfortunately, no. The `markdown-toc` program only has an append option. We could evaluate other tools if you think it's a big deal.

```
Usage: markdown-toc [options] <input> 

  input:        The Markdown file to parse for table of contents,
                or "-" to read from stdin.

  -i:           Edit the <input> file directly, injecting the TOC at <!-- toc -->
                (Without this flag, the default is to print the TOC to stdout.)

  --json:       Print the TOC in JSON format

  --append:     Append a string to the end of the TOC

  --bullets:    Bullets to use for items in the generated TOC
                (Supports multiple bullets: --bullets "*" --bullets "-" --bullets "+")
                (Default is "*".)

  --maxdepth:   Use headings whose depth is at most maxdepth
                (Default is 6.)

  --no-firsth1: Include the first h1-level heading in a file

  --no-stripHeadingTags: Do not strip extraneous HTML tags from heading
                         text before slugifying
```

Comment (akostibas) [reply README.md]: I suppose we could just add an inline comment manually..

Comment (brainsik) [reply README.md]: I was thinking just a manual comment. You can also flip me the bird and I'll approve without the change. :-)

**Comment by akostibas** [comment]
No need for incivilities. Change made!

**Pull request merged**